src/Resources/ShipmentOptions.php

```diff
@@ -50,6 +50,10 @@ public function setDefaultOptions(): self
         $this->package_type = PackageType::PACKAGE;
         $this->delivery_type = DeliveryType::STANDARD;
         $this->only_recipient = false;
+        $this->insurance = [
+            'amount' => 0,
+            'currency' => 'EUR'
+        ];
 
```

Comment:
Set it to an empty array here. 

In the `toArray` method the `insurance` attribute should be filtered out with the `reject` method on the collection class when it is empty.